### PR TITLE
Add LoganSquare config

### PR DESCRIPTION
**Pull request by Jawnnypoo**
Its a great Gson alternative!
https://github.com/bluelinelabs/LoganSquare

### CODE REVIEW

**Comment by mikepenz** [comment]
@Jawnnypoo oh thank you

File: library/src/main/res/values/library_logansquare_strings.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="define_int_LoganSquare">year;owner</string>
+    <string name="library_AboutLibraries_author">BlueLine Labs, Inc.</string>
```
Comment (mikepenz) [inline library/src/main/res/values/library_logansquare_strings.xml]: you have a copy paste error here. should be library_LoganSquare too

**Comment by Jawnnypoo** [comment]
Good catch on Copy paste. Should be good now!

**Comment by mikepenz** [comment]
Looks fine now. Thanks

**Pull request merged**